### PR TITLE
[opendaylight] collect more logs and puppet config

### PR DESCRIPTION
**Pull request by pmoravec**
collect puppet-generated config, container and docker logs

Resolves: #1171

Signed-off-by: Pavel Moravec <pmoravec@redhat.com>

---
Please place an 'X' inside each '[]' to confirm you adhere to our [Contributor Guidelines](https://github.com/sosreport/sos/wiki/Contribution-Guidelines)

- [X] Is the commit message split over multiple lines and hard-wrapped at 72 characters?
- [X] Is the subject and message clear and concise?
- [X] Does the subject start with **[plugin_name]** if submitting a plugin patch or a **[section_name]** if part of the core sosreport code?
- [X] Does the commit contain a **Signed-off-by: First Lastname <email@example.com>**?

### CODE REVIEW

File: sos/plugins/opendaylight.py [hunk 1]
```diff
@@ -24,15 +24,28 @@ class OpenDaylight(Plugin, RedHatPlugin):
 
     packages = ('opendaylight',)
 
+    var_puppet_gen = "/var/lib/config-data/puppet-generated/opendaylight"
+
     def setup(self):
-        self.add_copy_spec("/opt/opendaylight/etc/")
+        self.add_copy_spec([
```
Comment (bmr-cymru) [inline sos/plugins/opendaylight.py]: Wrong indentation level for this style of bracing. Should be:

```python
    self.add_copy_spec([
        "/opt/blah/blah",
        self.var_puppet_gen + "/opt/blah/blah"
    ])
```

File: sos/plugins/opendaylight.py [hunk 2]
```diff
@@ -24,15 +24,28 @@ class OpenDaylight(Plugin, RedHatPlugin):
...
+                            "/opt/opendaylight/etc/",
+                            self.var_puppet_gen + "/opt/opendaylight/etc/",
+                           ])
 
         self.limit = self.get_option("log_size")
         if self.get_option("all_logs"):
-            self.add_copy_spec("/opt/opendaylight/data/log/",
+            self.add_copy_spec([
```
Comment (bmr-cymru) [inline sos/plugins/opendaylight.py]: Same

File: sos/plugins/opendaylight.py [hunk 3]
```diff
@@ -24,15 +24,28 @@ class OpenDaylight(Plugin, RedHatPlugin):
...
+                                "/opt/opendaylight/data/log/",
+                                "/var/log/containers/opendaylight/",
+                               ],
                                sizelimit=self.limit)
         else:
-            self.add_copy_spec("/opt/opendaylight/data/log/*log",
+            self.add_copy_spec([
+                                "/opt/opendaylight/data/log/*.log*",
```
Comment (bmr-cymru) [inline sos/plugins/opendaylight.py]: Same

**Comment by pmoravec** [comment]
Updated.

**Pull request closed without merging**